Automattic/pocket-casts-android
# Playback 2023 - Epilogue story

| 📘 Part of: #1463 | 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|:---:|

## Description
Adds the 2023 visual for the Epilogue Story.

## Testing Instructions
1. Make sure you're logged in to an account that has a few episodes listened
3. Go to Profile
4. Tap the EOY image
5. Skip to the last story
5. ✅ Check that the story looks good
6. Tap "Play again"
7. ✅ You should go back to the first story
8. Give Patreon or Plus to your account and go to the last story again
9. ✅ The background should be slightly gold-ish

## Screenshots or Screencast 
Free user | Paid user
|---|---|
|<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/d84f0a6a-b63c-453d-8740-9291fce51748"/> | <img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/30c0bfc4-b640-4fed-83f8-ae81c9ca941d"/>|

## Checklist
 
#### I have tested any UI changes...
